tests/integration/components/lt-body-test.js

```diff
@@ -148,16 +148,20 @@ test('row expansion - multiple', function(assert) {
 });
 
 test('row actions', function(assert) {
-  assert.expect(2);
+  assert.expect(4);
 
   this.set('table', new Table(Columns, createUsers(1)));
   this.on('onRowClick', (row) => assert.ok(row));
   this.on('onRowDoubleClick', (row) => assert.ok(row));
-  this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions onRowClick=(action 'onRowClick') onRowDoubleClick=(action 'onRowDoubleClick')}}`);
+  this.on('onRowMouseEnter', (row) => assert.ok(row));
+  this.on('onRowMouseLeave', (row) => assert.ok(row));
+  this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions onRowClick=(action 'onRowClick') onRowDoubleClick=(action 'onRowDoubleClick') onRowMouseEnter=(action 'onRowMouseEnter') onRowMouseLeave=(action 'onRowMouseLeave')}}`);
```

Comment:
```js
await this.render(/* ... */);
```

Use `await` here please. It does not have any effect yet with the current `ember-qunit` version that's installed on your branch, but when https://github.com/offirgolan/ember-light-table/pull/531 is merged, it will. Please refer to [RFC 232](https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md#changes-from-current-system) for more info.